pages/linux/matchpathcon.md

```diff
@@ -0,0 +1,17 @@
+# matchpathcon
+
+> Lookup the persistent SELinux security context setting of a path.
+> See also: `semanage-fcontext`, `secon`, `chcon`, `restorecon`.
+> More information: <https://manned.org/man/matchpathcon.8>.
+
+- Lookup the persistent security context setting of a path:
+
+`matchpathcon {{/path/to/file}}`
```

Comment:
```suggestion
- Lookup the persistent security context setting of an absolute path:

`matchpathcon {{/path/to/file}}`
```

Does it have to be an absolute path? Otherwise, we could do this:

```suggestion
- Lookup the persistent security context setting of a path:

`matchpathcon {{path/to/file}}`
```

Reply:
Technically it's just a plain regex match, so neither form is technically correct.

Practically though, all policy rules are written with absolute path in mind. Non-absolute regexes would be a pretty big security loophole IMO.

So I prefer the first form.